lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -1156,11 +1156,10 @@ SmallVector<unsigned> DotOperandEncodingAttr::getWarpsPerCTA() const {
 }
 SmallVector<unsigned> DotOperandEncodingAttr::getWarpOrder() const {
   // FIXME(Lezcano): Preexisting. Do we want to have this path at all?
-  if (mlir::isa<AMDMfmaEncodingAttr>(getParent())) {
+  if (mlir::isa<AMDMfmaEncodingAttr, AMDWmmaEncodingAttr, BlockedEncodingAttr>(
```

Comment:
I have a feeling that this is true for all MMA encodings, but I need an conformation from someone.
@lezcano can you help?